src/Dev/Deprecation.php

```diff
@@ -245,6 +301,19 @@ public static function shouldShowForCli(): bool
         return Deprecation::$shouldShowForCli;
     }
 
+    /**
+     * If true, deprecation warnings will be shown for deprecated code which is called by core Silverstripe modules.
+     */
+    public static function getShowCalledFromSupportedCodeNotices(): bool
+    {
+        return Deprecation::$showCalledFromSupportedCodeNotices;
+    }
+
+    public static function setShowCalledFromSupportedCodeNotices(bool $value): void
```

Comment:
Add PHPDoc to this public method please.

Reply:
Done